caddy/staging.ocaml.org

```diff
@@ -1,3 +1,16 @@
 v3c.ocaml.org, staging.ocaml.org {
+	handle /host/metrics {
+		basic_auth bcrypt {
+			prometheus {{ prometheus_password }}
+		}
+		uri strip_prefix /host
+		reverse_proxy 172.17.0.1:9090
+	}
+	handle /metrics {
+		basic_auth bcrypt {
+			prometheus {{ prometheus_password }}
+		}
+		reverse_proxy www:8080
+	}
```

Comment:
```suggestion
```
There is no target for this nor any scrape configuration for this location.  This block would allow metrics to be scrapped directly from the application but it does not provide any.  Ocurrent provides metrics automatically.

Reply:
I see! This was cargo culting on my part. Thanks for the correction!